assets/scss/base/_buttons.scss

```diff
@@ -60,8 +63,12 @@ button, .button {
   color: $link-colour;
   border: none;
 
-  &:hover, 
-  &:focus, 
+  &.button--link-table {
```

Comment:
Need to review nesting here but might be dependant on uncoupling `.button` and `.button--link`